api/src/main/java/com/okta/sdk/client/ClientBuilder.java

```diff
@@ -306,9 +312,59 @@
      *
      * @param authenticationScheme the type of authentication to be used for communication with the Okta API server.
      * @return the ClientBuilder instance for method chaining
+     *
+     * @deprecated since 0.6.0 use {@link #setAuthorizationMode(AuthorizationMode)} to indicate the authentication scheme.
      */
+    @Deprecated
     ClientBuilder setAuthenticationScheme(AuthenticationScheme authenticationScheme);
 
+    /**
+     * Allows specifying an authorization mode directly instead of relying on the
+     * default location + override/fallback behavior defined in the {@link ClientBuilder documentation above}.
```

Comment:
the "documentation above" will not point to anything for users viewing the Javadoc from an IDE